SynthSeg/metrics_model.py

```diff
@@ -99,8 +99,9 @@ def __call__(self, gt, pred):
         ) / (tf.keras.backend.sum(weights) * self._n_labels)
 
 
-class DiceLoss:
+class DiceLoss(tf.keras.losses.Loss):
```

Comment:
Could you do the same for the `WeightedL2Loss`, in case we decide to apply this loss for the training?